### PR TITLE
XW-1893 | Add strings for global navigation

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: i18n/en/design-system.json [hunk 1]
```diff
@@ -43,5 +43,25 @@
...
+  "global-navigation-fandom-overview-community-central": "Community Central",
+  "global-navigation-fandom-overview-explore-wikis": "Explore Wikis",
+  "global-navigation-fandom-overview-link-vertical-games": "Games",
+  "global-navigation-fandom-overview-link-vertical-movies": "Movies",
+  "global-navigation-fandom-overview-link-vertical-tv": "TV",
+  "global-navigation-fandom-overview-trending-wikis": "Trending Wikis",
+  "global-navigation-fandom-overview-wikis": "Wikis",
+  "global-navigation-notifications-title": "Notifications",
+  "global-navigation-profile-title": "Profile",
+  "global-navigation-search-placeholder": "Search",
```
Comment (kvas-damian) [inline i18n/en/design-system.json]: We use few different strings for placeholders in search. I can't find them here.

**Comment by kvas-damian** [comment]
I see few string missing that are present on designs:
- register button description
- Link to My wall in user menu
- search ghost text in active state

File: i18n/en/design-system.json [hunk 2]
```diff
@@ -43,5 +43,25 @@
   "global-footer-licensing-and-vertical-description-param-vertical-music": "Music",
   "global-footer-licensing-and-vertical-description-param-vertical-movies": "Movies",
   "global-footer-full-site-link": "View Full Site",
-  "global-footer-mobile-site-link": "View Mobile Site"
+  "global-footer-mobile-site-link": "View Mobile Site",
+  "global-navigation-fandom-logo-title": "Fandom",
+  "global-navigation-fandom-overview-community-central": "Community Central",
+  "global-navigation-fandom-overview-explore-wikis": "Explore Wikis",
+  "global-navigation-fandom-overview-link-vertical-games": "Games",
```
Comment (kvas-damian) [inline i18n/en/design-system.json:52]: I'm not sure why do we translate vertical name so many times. Why can't we reuse it?

**Comment by kvas-damian** [comment]
I think it'd be great if message keys are the same as key used in API: https://github.com/Wikia/app/blob/dev/includes/wikia/api/DesignSystemApiController.class.php

We can change them here or there - it doesn't matter for me.

File: i18n/en/design-system.json (hunk 2 above)
Comment (hakubo) [reply i18n/en/design-system.json:52]: agree!

how would you name the key?
vertical-games
vertical-tv
?

and we'll have to update footer...

**Comment by hakubo** [comment]
@kvas-damian what do you mean by : Link to My wall in user menu
I can't find it either in doc or in Zeplin

File: i18n/en/design-system.json (hunk 2 above)
Comment (kvas-damian) [reply i18n/en/design-system.json:52]: what about `vertical-tv-name`?

**Comment by kvas-damian** [comment]
I'm not surewhy it's not defined there but I definitely see this string on muppet wiki logged in state:
zaznaczenie_184

File: i18n/en/design-system.json [hunk 3]
```diff
@@ -43,5 +43,32 @@
...
+  "global-navigation-anon-register-description": "Don't have an account?",
+  "global-navigation-anon-sign-in": "Sign In",
+  "global-navigation-create-wiki-link-start-wikia": "Start a Wiki",
+  "global-navigation-fandom-overview-link-vertical-games": "Games",
+  "global-navigation-fandom-overview-link-vertical-movies": "Movies",
+  "global-navigation-fandom-overview-link-vertical-tv": "TV",
+  "global-navigation-notifications-title": "Notifications",
+  "global-navigation-search-cancel": "Cancel",
+  "global-navigation-search-placeholder-fandom": "Search Stories and Wikis...",
+  "global-navigation-search-placeholder-in-wiki": "Search __sitename__",
```
Comment (kvas-damian) [inline i18n/en/design-system.json]: Are we sure about this one? We finish other active placeholders with three dots. So we we have an exception here.
The same pattern is followed on production.

Comment (hakubo) [reply i18n/en/design-system.json]: I asked this question in the doc.

**Comment by rogatty** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**